src/site/cookies/index.njk

```diff
@@ -0,0 +1,101 @@
+---
+title: Cookies on the MOD.UK Design System
+description: How we use cookies to make the MOD.UK Design System work and collect information about how you use this website.
+tags: footer
+order: 300
```

Comment:
Hmm the order is inconsistent in Figma, @smather76 commented on the Accessibility statement saying he expected it to be first so might need to clarify